packages/myst-cli/src/build/build.ts

```diff
@@ -126,6 +126,7 @@ function extToKind(ext: string): string {
 }
 
 export async function build(session: ISession, files: string[], opts: BuildOpts) {
+  await session.loadPlugins();
```

Comment:
@fwkoch is there anywhere else that we would want to load the plugins? Right now this is in build and start.

Reply:
Hmm, good question. I see two alternatives:

1. We could load `plugins` immediately when handling frontmatter (like at the end of validation or something) - this seems like the most mindless approach: To do anything, we must read frontmatter, so plugins will definitely be loaded. The drawback of this approach is we would do a bunch of unnecessary loading (e.g. `myst clean` reads frontmatter, really shouldn't need to load plugins).
2. Load plugins immediately before we do any processing at the `loadFile` level (I think this would be the place to do it - this is where we apply roles/directives, then transforms come immediately after)

The places you have it also work - they are just at a `cli`-centric level and maybe don't consider others using exported functions individually...

Reply:
I have moved this over to `loadFile`. Great idea. :)